src/native/fs/fs_napi.cpp

```diff
@@ -1412,6 +1412,7 @@ fs_napi(Napi::Env env, Napi::Object exports)
     exports_fs["S_IFLNK"] = Napi::Number::New(env, S_IFLNK);
     exports_fs["DT_DIR"] = Napi::Number::New(env, DT_DIR);
     exports_fs["DT_LNK"] = Napi::Number::New(env, DT_LNK);
+    exports_fs["PLATFORM_IOV_MAX"] = Napi::Number::New(env, sysconf(_SC_IOV_MAX));
```

Comment:
I just wonder if this is a portable way to get the value.
I thought that getting it from `IOV_MAX` in `<limits.h>` might be more portable? WDYT?

Reply:
sure, I replaced sysconf(_SC_IOV_MAX) with IOV_MAX of limits.h